docs/data/external-data/abares-clum-2020/_specifications.yaml

```
@@ -0,0 +1,13 @@
+# Bands
+
+bands:
+  footnotes: null
+  bands_table:
+    - name: alum_class
+      aliases: []
+      resolution: 50 metres
+      crs: "EPSG:3577"
+      units: Metres
```

Either this, or leave this one empty

```suggestion
      units: Classification
```